### DISCUSSION TITLE
Working With Test Results form Multiples XML files

### DISCUSSION THREAD

**Discussion by jitesha653** [EvotecIT/PSWriteHTML#377] [General]
How do we uses nested tab based on the xml files output. For example, if i have  10 xml files in one folder with different output and the below 3 tabs are common for all the test results 
![image](https://user-images.githubusercontent.com/107789153/232031105-87f200ae-6657-47b0-8389-dbb385e268f5.png)

can you help me with a sample solution
What i need to achieve is, if i click First Test 1 i should get results related to test 1 only
if i click  Second Test 2, i should get results related to test 2 only 
and so on

**Comment by PrzemyslawKlys** [comment]
From the screenshot above it seems like you have it figured out - not sure where you need actual help?

**Comment by jitesha653** [comment]
Need to understand how do I use the data of the diffrent XML on every tab
Maybe by using for each loop
Not able to figure out how to use it

**Reply by jitesha653** [reply]
I have figured it out and working on it
